download.mdwn

```diff
@@ -54,6 +54,7 @@ A list of of old releases [[is available here|releases]].
 * [openSUSE](http://download.opensuse.org/repositories/X11:/windowmanagers/)
 * [PLD Linux](http://pld-linux.org)
 * [Mageia](http://mageia.madb.org/package/show/application/0/name/awesome)
+* [OpenIndiana](http://pkg.openindiana.org/hipster/en/search.shtml?token=awesome&action=Search)
```

Comment:
I'm afraid it cannot. Maybe it is better to link to the OpenIndiana site? It's up to you. I just wanted you guys to know that Awesome is available as a package in OpenIndiana.